core/src/main/java/org/apache/cxf/interceptor/FaultOutInterceptor.java

```diff
@@ -25,6 +25,7 @@
 import java.util.logging.Logger;
 
 import javax.xml.stream.XMLStreamWriter;
+import javax.xml.ws.soap.SOAPFaultException;
```

Comment:
This makes core depend on SAAJ which is definitely not what we'd want to do.   Let me take a look at the test case to see if there is a different way to handle this from the jaxws fault interceptors.